Update changelog for v1.1.0 (#286)
NordicSemiconductor/pc-nrfconnect-quickstart: Changelog.md

-   Updated modem firmware packages for nRF9151 DK and nRF9161 DK to 2.0.2.
-   Updated modem firmware packages for nRF9160 DK to 1.3.7.

### Fixed

> -   Some cases when network core programming for nRF5340 DK was failing.

Reviewer: ```suggestion
-   Some cases when programming the network core on the nRF5340 DK would fail.
```
